tests/catch2/catch.hpp

```diff
@@ -0,0 +1,17959 @@
+/*
+ *  Catch v2.13.7
+ *  Generated: 2021-07-28 20:29:27.753164
```

Comment:
I believe I wanted to add it properly as a dependency, but I think the problem was that the CMake version does not support FetchContent (and my knowledge of CMake is too limited to be able to do it otherwise), so I added the one necessary file manually. I believe it is done in the same way in tests for dg.